src/components/Dashboard/Wrappers/Category/useCategoryActions.tsx

```diff
@@ -187,6 +187,10 @@ export const useCategoryActions = (configName: string | undefined, category: Cat
       (previous) => {
         const currentItem = previous.categories.find((x) => x.id === category.id);
         if (!currentItem) return previous;
+
+        const currentWrapper = previous.wrappers.find((x) => x.position === currentItem?.position);
```

Comment:
still the same, did you forget to push?

Reply:
Yes sorry, commited it on dev first :joy: